chart/templates/worker-deployment.yaml

```diff
@@ -105,14 +129,46 @@ spec:
         {{- if $root.Values.clamav.enabled  }}
         - name: clamav
```

Comment:
Why not add clamav configuration to the parent values.yaml to ensure all deployments are using the correct values?

Reply:
I don't understand - the clamav resource definition is in the parent helm chart's template directory - which is a significantly better place for items that we want to ensure are set up consistently between environments - we have some values that are set in the parent helm charts values.yaml, because we want those values to be configurable at deploy time to the different environments

Reply:
we do not want the resource definition (i.e. the entire configuration block) - to be overwritten at deploy time - we instead want consistency. If someone wants a major change to the way we set up clamav - that should be a branch off the master branch holding the helm chart and then thoroughly test it before merging the parent helm chart into master.